SDK/AppCenterPush/Microsoft.AppCenter.Push.Android.Bindings/Microsoft.AppCenter.Push.Android.Bindings.csproj

```diff
@@ -9,7 +9,7 @@
     <OutputType>Library</OutputType>
     <RootNamespace>Microsoft.AppCenter.Push.Android.Bindings</RootNamespace>
     <AssemblyName>Microsoft.AppCenter.Push.Android.Bindings</AssemblyName>
-    <TargetFrameworkVersion>v8.0</TargetFrameworkVersion>
+    <TargetFrameworkVersion>v9.0</TargetFrameworkVersion>
```

Comment:
Was that necessary? If yes we need to say it's a breaking change. We used to document mono target updates.